activerecord/lib/active_record/dynamic_includes.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+module ActiveRecord
+  # DynamicIncludes is a module that can be included to allow for dynamic loading of
```

Comment:
I'm kind of confused by this sentence. Is the intention for applications to be required to include this? I don't think that's necessary since this module is autoloaded in activerecord/lib/active_record.rb. If apps aren't supposed to include this or use it directly, it should be `no-doc`.